latest_migrations.manifest

```diff
@@ -1,6 +1,6 @@
 admin: 0003_logentry_add_action_flag_choices
 auth: 0011_update_proxy_permissions
 contenttypes: 0002_remove_content_type_name
-posthog: 0057_action_updated_at
+posthog: 0058_auto_20200605_1021
```

Comment:
This should remain unchanged because I think you originally added a migration and then pulled it later

Reply:
Fixed